src/app/econ/supply-demand/lesson/SupplyDemandLesson.tsx

```diff
@@ -0,0 +1,807 @@
+        lots of people want it!{" "}
+      </p>
+      <ColorBox color="blue">
+        <p>
+          The law of demand states that{" "}
+          <Emphasize>
+            as price goes up, the amount that people want to buy
+          </Emphasize>{" "}
+          (Quantity Demanded) <Emphasize> goes down</Emphasize>, and vice versa.
+        </p>
+        <div>
+          <Emphasize>
+            <u>Reasons: </u>
+          </Emphasize>
+          <ul>
+            <li>
+              <Collapsible
+                ExampleContent={
+                  <>
+                    <Emphasize>
+                      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;👉 Example:{" "}
+                    </Emphasize>{" "}
+                    If ice cream is too expensive, they will buy a candy bar
+                    instead
+                  </>
+                }
+              >
+                <Emphasize>- Substitution effect: </Emphasize> Consumers switch
+                to alternatives.
+              </Collapsible>
+            </li>
+            <li>
+              <Collapsible
+                ExampleContent={
+                  <>
+                    <Emphasize>
+                      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;👉 Example:{" "}
+                    </Emphasize>{" "}
+                    If they have more money to spend, they will get fine dining
+                    instead of McDonald's.
+                  </>
+                }
+              >
+                <Emphasize>- Income effect: </Emphasize> Budget constraints
+                change purchasing power
+              </Collapsible>
+            </li>
+            <li>
+              <Collapsible
+                ExampleContent={
+                  <>
+                    <Emphasize>
+                      &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;👉 Example:{" "}
+                    </Emphasize>{" "}
+                    If they have more and more ice cream, they will be less and
+                    less satisfied with each additional ice cream.
+                  </>
+                }
+              >
+                <Emphasize>- Diminishing </Emphasize>
+                <a
+                  href="https://pincs.stanford.edu/pickcode-dev/econ/marginal-utility/?slide=1"
+                  style={{ color: "blue", textDecoration: "underline" }}
+                >
+                  marginal utility:
+                </a>{" "}
+                The satisfaction from buying more of the same thing decreases.{" "}
+              </Collapsible>
+            </li>
+          </ul>
+        </div>
+      </ColorBox>
+      <ColorBox color="purple">
+        <p>
+          <Emphasize>
+            What is the difference between demand and quantity demanded?
+          </Emphasize>
+        </p>
+        <p>
+          Demand is the entire relationship between prices and how much people
+          want at each price. Quantity demanded is how much people want at one
```

Comment:
nit: I'd somehow emphasize the key terms (Demand, Quantity demanded) here, since they're the definitions from the above question